index.ts

```diff
@@ -740,11 +739,14 @@ commander
       open(url);
       process.exit(0);
     }
-    const logger = new logging({
-      projectId,
-    });
-    return logger.getEntries().then(printLogs).catch((err: any) => {
-      console.error(ERROR.LOGS_UNAVAILABLE);
+    getAPICredentials(async () => {
+      const logger = google.logging({version: 'v2', auth: oauth2Client}) as Logging;
```

Comment:
Put `const logger` here:
https://github.com/google/clasp/blob/master/index.ts#L61

Reply:
On a related note, should we do this for `const drive` as well? https://github.com/google/clasp/blob/439772ad83bd061cfe14a7f4d6b5289345aab20f/index.ts#L291